### PR TITLE
Port changes from 5.6.3 release

### PR DESCRIPTION
**Pull request by aarongranick-okta**
### Issue:

- [OKTA-395447](https://oktainc.atlassian.net/browse/OKTA-395447)

### CODE REVIEW

**Review by worldleelavivat-okta — APPROVED** [review]
my change looks good

**Review by TianGan-okta — APPROVED** [review]

File: test/unit/spec/v2/ion/i18nTransformer_spec.js [hunk 1]
```diff
@@ -1825,6 +1825,41 @@ describe('v2/ion/i18nTransformer', function() {
     ],
   ])('should render correct error message by get key from API: %s', (key, error, expectedMessage) => {
     Bundles.login = originalLoginBundle;
+    expect(Bundles.login[key]).toBeUndefined();
```
Comment (sherizada-okta) [inline test/unit/spec/v2/ion/i18nTransformer_spec.js:1899]: Not my change

File: test/unit/spec/v2/ion/i18nTransformer_spec.js [hunk 2]
```diff
@@ -1825,6 +1825,41 @@ describe('v2/ion/i18nTransformer', function() {
...
     expect(getMessage(error)).toEqual(expectedMessage);
   });
+
+  it('returns an override key when present in the override map', () => {
```
Comment (sherizada-okta) [inline test/unit/spec/v2/ion/i18nTransformer_spec.js:1903]: this test case was not my change https://github.com/okta/okta-signin-widget/commit/baf72db7a55bda231ade1fc1c46d14dc62df473d

File: test/unit/spec/v2/ion/i18nTransformer_spec.js (hunk 1 above)
Comment (sherizada-okta) [inline test/unit/spec/v2/ion/i18nTransformer_spec.js:1899]: this was not my change https://github.com/okta/okta-signin-widget/commit/baf72db7a55bda231ade1fc1c46d14dc62df473d

**Review by haishengwu-okta — APPROVED** [review]

**Comment by aarongranick-okta** [comment]
Closed in favor of https://github.com/okta/okta-signin-widget/pull/1953

**Pull request closed without merging**